deployment/istio-configs/sleep-hsm.yaml

```diff
@@ -25,7 +25,7 @@ rules:
   resources: ["certificatesigningrequests", "certificatesigningrequests/approval", "certificatesigningrequests/status"]
   verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
 - apiGroups: ["certificates.k8s.io"]
-  resourceNames: [tcsclusterissuer.tcs.intel.com/*]
+  resourceNames: ["clusterissuers.cert-manager.io/*", "tcsclusterissuer.tcs.intel.com/*", "tcsissuer.tcs.intel.com/*"]
```

Comment:
I think we should go through mTLS workload and gateway use cases on cert-manager and tcs before this configuration change.

Reply:
This yaml can work with cert manager, I'll double check integration with TCS today.